root/edit/details/AddRelationshipType.js

```diff
@@ -0,0 +1,169 @@
+/*
+ * @flow strict-local
+ * Copyright (C) 2020 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+
+import IntentionallyRawIcon from '../components/IntentionallyRawIcon';
+import Cardinality from '../../static/scripts/common/components/Cardinality';
+import OrderableDirection
+  from '../../static/scripts/common/components/OrderableDirection';
+import {ENTITY_NAMES} from '../../static/scripts/common/constants';
+import expand2react from '../../static/scripts/common/i18n/expand2react';
+
+type AddRelationshipTypeEditT = {
+  ...EditT,
+  +display_data: {
+    +attributes: $ReadOnlyArray<{
+      ...LinkTypeAttrTypeT,
+      +typeName: string,
+    }>,
+    +child_order: number,
+    +description: string | null,
+    +documentation: string | null,
+    +entity0_cardinality?: number,
+    +entity0_type: CoreEntityTypeT,
+    +entity1_cardinality?: number,
+    +entity1_type: CoreEntityTypeT,
+    +link_phrase: string,
+    +long_link_phrase: string,
+    +name: string,
+    +orderable_direction?: number,
+    +reverse_link_phrase: string,
+  },
+};
+
+type Props = {
+  +edit: AddRelationshipTypeEditT,
+};
+
+const AddRelationshipType = ({edit}: Props): React.Element<'table'> => {
+  const display = edit.display_data;
+  const entity0Type = ENTITY_NAMES[display.entity0_type]();
+  const entity1Type = ENTITY_NAMES[display.entity1_type]();
+  const rawIconSection = (
+    <>
+      {' '}
+      <IntentionallyRawIcon />
+    </>
+  );
+
+  return (
+    <table className="details add-relationship-type">
+      <tr>
+        <th>{addColonText(l('Name'))}</th>
+        <td>
+          {display.name}
+          {rawIconSection}
+        </td>
+      </tr>
+
+      <tr>
+        <th>{addColonText(l('Description'))}</th>
+        <td>
+          {nonEmpty(display.description)
+            ? (
+              <>
+                {display.description}
+                {rawIconSection}
+              </>
+            ) : lp('(none)', 'description')}
+          {}
+        </td>
+      </tr>
+
+      <tr>
+        <th>{l('entity0 type:')}</th>
```

Comment:
- `entity0` should not be translated as it is a placeholder;
- Not starting for a capital letter may mislead translator too;
- `:` can be saved here.

```suggestion
        <th>{addColonText(l('Type of {entity0}'))}</th>
```

Reply:
This just displays `Type of {entity0}` as the actual string, I assume that's not the intention? Same for the other {entity0/1} ones here. Making the other changes for now but without the {}.

Reply:
To make the intention more obvious and prevent localization of the placeholder, we can even use:
```jsx
<th>{addColonText(l('Type of {entity_placeholder}', {
  entity_placeholder: '{entity0}',
}))}</th>
```

Reply:
Huh. Ok, I didn't consider that. Honestly, to me it just feels like something broke :/

What about this?

```
<th>{addColonText(l('Type of {entity_placeholder}', {
  entity_placeholder: 'entity0',
}))}</th>
```

It should still make it so that it doesn't get translated but to users it won't look like the string broke. And it allows us to only translate each header once, which I do like.

Reply:
It’s not _broken_, it is _intentionally left as it was originally entered (untranslated, unformatted)_ which is explained thanks to the :information_source: information icons you added. 

The full placeholder `{entity0}` is printed in the linking phrases as-is, so I feel like printing the full placeholder here makes the relationship more obvious. Maybe just adding the tag `code` would help to make it feel less broken?

Reply:
I guess that's a bit better, I'm still not a fan but I can live with it.

![Screenshot from 2020-10-14 21-15-51](https://user-images.githubusercontent.com/1069224/96030203-60e3ea80-0e64-11eb-9bb0-9d39dc373d7c.png)

Reply:
Link phrases should keep `{entity0}` as [suggested above](
https://github.com/metabrainz/musicbrainz-server/pull/1636#discussion_r470461998) otherwise there is no point to have it in type label indeed.

Reply:
As I mentioned as an answer to that, the phrases just don't include the placeholders in most cases. I could just add them by hand, but then it would break when they *do* include them...

Reply:
Maybe test link phrases with a RegExp to avoid such breakage?

Reply:
Ok, made that change, hopefully this seems ready now :)